apps/formbricks-com/pages/in-app-survey/index.tsx

```diff
@@ -0,0 +1,312 @@
+  },
+  {
+    question:
+      "How does Formbricks compare to other micro survey software in terms of features and flexibility?",
+    answer:
+      "Formbricks offers a comprehensive suite of features for embedded surveys, in-app feedback, and micro surveys. For see the speed development, have a look at the Formbricks repository on GitHub linked in the Footer. In case you're missing something, just let us know and we'll build it.",
+  },
+  {
+    question: "Is Formbricks GDPR-compliant for use as an in-app survey tool and embedded survey platform?",
+    answer:
+      "Yes, Formbricks is fully GDPR and CCPA compliant. It's a reliable choice for businesses seeking an in-app survey tool which handles potentially personalized data. Hosted in Frankfurt, Germany, and developed by a German company, it ensures the highest standards of data protection.",
+  },
+  {
+    question: "Can Formbricks help in conducting micro surveys within a mobile app?",
+    answer: "Currently, we do not offer SDKs for mobile apps yet. However, this is on the roadmap for 2024.",
+  },
+  {
+    question: "What are the best tools for creating an app survey?",
+    answer:
+      "For creating app surveys, Formbricks is among the top tools. As an open source product, we keep developer requirements at heart. However, the UX of Formbricks is designed also support marketers, researchers and sales reps in their work.",
+  },
+  {
+    question: "How can I implement an in-app survey effectively?",
+    answer:
+      "To implement an in-app survey effectively, use Formbricks to embed surveys directly into your application. In-product research has higher conversion and completion rates than any other form of surveying.",
+  },
+  {
+    question: "What is a micro survey and how can I use it with Formbricks?",
+    answer:
+      "A micro survey is a short, focused survey designed to capture quick insights. With Formbricks, you can easily create and embed these micro surveys into your website or app, enhancing the user experience and obtaining precise feedback.",
+  },
+  {
+    question: "Are embedded surveys more effective for user engagement?",
+    answer:
+      "Yes, embedded surveys, like those created with Formbricks, are highly effective for user engagement. They blend naturally with the app or website, encouraging more users to participate and share their insights without leaving the platform.",
+  },
+  {
+    question: "What advantages does Formbricks offer for in-app survey tools?",
+    answer:
+      "Formbricks offers numerous advantages for in-app surveys, including easy integration, real-time analytics, customizable survey templates, and micro survey capabilities. It's a powerful tool for enhancing user engagement and feedback collection.",
+  },
+];
+
+export default function LinkSurveyPage() {
+  return (
+    <LayoutLight
+      title="In app Surveys, Open Source"
```

Comment:
I think `in-App Surveys` (with a dash) is more common. We can choose either `In-App` or `In-app` but I think we should be consistent.